### DISCUSSION TITLE
“下载”界面新增下载光影和材质包

### DISCUSSION THREAD

**Discussion by LBacteria** [Meloong-Git/PCL#501] [提问] [answered]
如题，在“下载”界面原有的“游戏组件”，“mod”，“整合包”的基础上增加“光影”以及“材质包”

**Comment by WForst-Breeze** [comment, marked as answer]
您好，很高兴为您解答\(￣︶￣*\))
1. 光影：由于下载源问题，不会制作。见 #396 
2. 资源包（材质包）：见 #45 ，请于功能投票界面投票qwq
